stack/dashboard/base/builder.sh

```diff
@@ -171,6 +271,7 @@ mv ./package.json.tmp ./package.json
 /bin/bash ./bin/opensearch-dashboards-plugin remove observabilityDashboards --allow-root
 /bin/bash ./bin/opensearch-dashboards-plugin remove securityAnalyticsDashboards --allow-root
 /bin/bash ./bin/opensearch-dashboards-plugin remove searchRelevanceDashboards --allow-root
+/bin/bash ./bin/opensearch-dashboards-plugin remove mlCommonsDashboards --allow-root
```

Comment:
Why?

Reply:
OpenSearch Dashboards 2.10.0 includes a plugin that is not desired according to this design https://github.com/wazuh/wazuh-dashboard-plugins/issues/5893 to be included in Wazuh dashboard 4.8.0. This removes it from the OpenSearch Dashboard base.

Reply:
Thanks, I didn't see the relationship between removing the plugin and the changes requested from the original issue.